perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -176,19 +375,121 @@ void MapBasedPredictionNode::objectsCallback(const TrackedObjects::ConstSharedPt
     }
 
     // For non-vehicle object
+    const auto & label = transformed_object.classification.front().label;
     if (
-      transformed_object.classification.front().label != ObjectClassification::CAR &&
-      transformed_object.classification.front().label != ObjectClassification::BUS &&
-      transformed_object.classification.front().label != ObjectClassification::TRAILER &&
-      transformed_object.classification.front().label != ObjectClassification::MOTORCYCLE &&
-      transformed_object.classification.front().label != ObjectClassification::TRUCK) {
-      PredictedPath predicted_path =
-        path_generator_->generatePathForNonVehicleObject(transformed_object);
-      predicted_path.confidence = 1.0;
-
+      label != ObjectClassification::CAR && label != ObjectClassification::BUS &&
+      label != ObjectClassification::TRAILER && label != ObjectClassification::MOTORCYCLE &&
+      label != ObjectClassification::TRUCK) {
```

Comment:
pls separate for each class.
example
```
if(
     label == ObjectClassification::CAR && label == ObjectClassification::BUS ) {
    foo;
} else if (label == ObjectClassification::PEDESTRIAN){
    foo;
} else  if (label == ObjectClassification::UNKNOWN){
    foo;
}
}
```

Reply:
@yukkysaito OK :+1: I fixed in https://github.com/autowarefoundation/autoware.universe/pull/922/commits/3b43997a42fa831c7e3a29a281faed6662e44043.